stable/mongodb/values.yaml

```diff
@@ -1,15 +1,20 @@
-## Bitnami MongoDB image version
-## ref: https://hub.docker.com/r/bitnami/mongodb/tags/
-##
 image:
+  ## Bitnami MongoDB registry
+  ##
   registry: docker.io
-  repository: bitnami/mongodb
```

Comment:
Why did you change this parameter name? By the way, it's not aligned with the `README.md`

Reply:
human mistake!